.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: go
+go:
+  - 1.4
+  - 1.5
+install:
+  - go get golang.org/x/tools/cmd/vet
+script:
+  - cd $TRAVIS_BUILD_DIR/multi-node/aws
```

Comment:
Oh, nevermind, godeps only exists within `multi-node/aws`, so build environment would have to be set up separately for any future code.